demo/node/node_inprocess.go

```diff
@@ -321,6 +314,6 @@ func (l *LocalNode) PrintLog() {
 		fmt.Printf("[-] Can't read logs at %s !\n\n", l.logPath)
 		return
 	}
-	os.Stdout.Write([]byte(buff))
+	os.Stdout.Write(buff)
```

Comment:
I couldn't find any places where we overwrite this. I think the intention was to avoid the `string(buff)`. I changed this to use `fmt.Printf()` instead.